From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Manuel Palenzuela <manuelpalenzuelamerino@gmail.com>
Date: Sat, 3 Apr 2021 17:53:44 +0200
Subject: [PATCH] Disable some extra features

This just disables some extra features the game has such as screenshots, hardware acceleration, etc.
---
 src/config.h | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/src/config.h b/src/config.h
index 698e7edf436461c475cbe4531ef8bf8d10456cd5..0a935eeb353141fcf4db3b4bcfb6925a6679df98 100644
--- a/src/config.h
+++ b/src/config.h
@@ -35,7 +35,7 @@ The authors of this program may be contacted at https://forum.princed.org
 
 // Enable or disable the SDL hardware accelerated renderer backend
 // Uses a software backend otherwise
-#define USE_HW_ACCELERATION
+//#define USE_HW_ACCELERATION
 
 // Enable or disable fading.
 // Fading used to be very buggy, but now it works correctly.
@@ -58,23 +58,23 @@ The authors of this program may be contacted at https://forum.princed.org
 //#define USE_COMPAT_TIMER
 
 // Enable quicksave/load feature.
-#define USE_QUICKSAVE
+//#define USE_QUICKSAVE
 
 // Try to let time keep running out when quickloading. (similar to Ctrl+A)
 // Technically, the 'remaining time' is still restored, but with a penalty for elapsed time (up to 1 minute).
 // The one minute penalty will also be applied when quickloading from e.g. the title screen.
-#define USE_QUICKLOAD_PENALTY
+//#define USE_QUICKLOAD_PENALTY
 
 #ifdef USE_QUICKSAVE // Replay relies on quicksave, because the replay file begins with a quicksave of the initial state.
 
 // Enable recording/replay feature.
-#define USE_REPLAY
+//#define USE_REPLAY
 
 #endif
 
 // Adds a way to crouch immediately after climbing up: press down and forward simultaneously.
 // In the original game, this could not be done (pressing down always causes the kid to climb down).
-#define ALLOW_CROUCH_AFTER_CLIMBING
+//#define ALLOW_CROUCH_AFTER_CLIMBING
 
 // Time runs out while the level ending music plays; however, the music can be skipped by disabling sound.
 // This option stops time while the ending music is playing (so there is no need to disable sound).
@@ -90,12 +90,12 @@ The authors of this program may be contacted at https://forum.princed.org
 #define USE_FAKE_TILES
 
 // Allow guard hitpoints not resetting to their default (maximum) value when re-entering the room
-#define REMEMBER_GUARD_HP
+//#define REMEMBER_GUARD_HP
 
 // Enable completely disabling the time limit. To use this feature, set the starting time to -1.
 // This also disables the in-game messages that report how much time is left every minute.
 // The elasped time is still kept track of, so that the shortest times will appear in the Hall of Fame.
-#define ALLOW_INFINITE_TIME
+//#define ALLOW_INFINITE_TIME
 
 
 // Bugfixes:
@@ -277,11 +277,11 @@ The authors of this program may be contacted at https://forum.princed.org
 #define USE_LIGHTING
 
 // Enable screenshot features.
-#define USE_SCREENSHOT
+//#define USE_SCREENSHOT
 
 // Automatically switch to keyboard or joystick/gamepad mode if there is input from that device.
 // Useful if SDL detected a gamepad but there is none.
-#define USE_AUTO_INPUT_MODE
+//#define USE_AUTO_INPUT_MODE
 
 #ifdef USE_TEXT // The menu won't work without text.
 
@@ -291,13 +291,13 @@ The authors of this program may be contacted at https://forum.princed.org
 #endif
 
 // Enable colored torches. A torch can be colored by changing its modifier in a level editor.
-#define USE_COLORED_TORCHES
+//#define USE_COLORED_TORCHES
 
 // Enable fast forwarding with the backtick key.
-#define USE_FAST_FORWARD
+//#define USE_FAST_FORWARD
 
 // Set how much should the fast forwarding speed up the game.
-#define FAST_FORWARD_RATIO 10
+//#define FAST_FORWARD_RATIO 10
 
 // Speed up the sound during fast forward using resampling.
 // If disabled, the sound is sped up by clipping out parts from it.
